lms/validation/authentication/_lti.py

```diff
@@ -130,4 +132,6 @@ def lti_user(self):
                 {"JWT": ["Invalid LTI1.3 params. Unknown application_instance."]}
             ) from err
 
-        return self._lti_user_service.from_auth_params(application_instance, kwargs)
+        return self._lti_user_service.from_lti_params(
+            application_instance, self.context["request"].lti_params
+        )
```

Comment:
I'll be honest and say I find this whole validation marshmallow schema mixed with authentication totally baffling.